utils_cv/detection/dataset.py

```diff
@@ -231,17 +231,14 @@ def split_train_test(
         Return
             A training and testing dataset in that order
         """
-        # TODO Is it possible to make these lines in split_train_test() a bit
-        # more intuitive?
-
         test_num = math.floor(len(self) * (1 - train_pct))
         indices = torch.randperm(len(self)).tolist()
 
         self.transforms = get_transform(train=True)
```

Comment:
I find this syntax quite confusing. Could rewrite to:
        train = Subset(self, indices[test_num:])
        train.dataset.transforms = get_transform(train=True)

        test = Subset(self, indices[: test_num + 1])
        test.dataset.transforms = get_transform(train=False)

Reply:
done